nvm-backstage/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright © 2018 Booz Allen Hamilton. All Rights Reserved.
```

Comment:
Instead of copying the base NVM Dockerfile and making some tweaks, I'd suggest making this use the origianl NVM image as a base image. That would reduce a lot of duplicated code.